cr8/run_crate.py

```diff
@@ -295,9 +298,15 @@ def start(self):
 
     def _set_addr(self, protocol, addr):
         log.info('{0:10}: {1}'.format(protocol.capitalize(), addr))
+        host, port = addr.rsplit(':', 1)
+        port = int(port)
+        self.addresses[protocol] = Address(host, port) 
         if protocol == 'http':
             self.http_host = addr
             self.http_url = 'http://' + addr
+        elif protocol == 'psql':
```

Comment:
I think we should not set `postgres_host` and `postgres_url` too. I'd consider `http_host` and `http_url` legacy and have `addresses` be the thing that API users should use. (Here there would also be the problem that we fix the users to `crate` which might not be the thing a client wants to have)